clipmenu

```diff
@@ -57,11 +57,25 @@ else
 fi
 
 [[ $chosen_line ]] || exit 1
-file=$cache_dir/$(cksum <<< "$chosen_line")
-[[ -f "$file" ]] || exit 2
+
+files=()
+while IFS= read -r line; do
+    file=$cache_dir/$(cksum <<< "$line")
+    files+=("$file")
+done <<< "$chosen_line"
+
+paste=()
+for file in "${files[@]}"; do
+    contents=$(cat "$file")
+    paste+=("$contents")
+done
+
+paste=$( IFS=$'\n'; echo "${paste[*]}" )
```

Comment:
This just replaces `${paste[0]}` and leaves everything else intact. Either way, it seems unnecessary -- just use cat to join the data.

Reply:
Will take a look.